doc/getting_started/getting_started.rst

```diff
@@ -94,6 +124,16 @@ To build an example application follow these steps:
       :build-dir: arduino_101
       :goals: build
 
+  On Linux/macOS you can also build with ``make`` instead of ``ninja``:
```

Comment:
You need to indent this paragraph and the following zephyr-app-commands with one extra space (one space each for '#', '.', and ' ')

Reply:
Fixed thanks